discord_bot_main.py

```diff
@@ -77,16 +82,18 @@ async def runAutoOctoFinder(self, message):
         # limitation.
         header_embed = discord.Embed()
         header_embed.add_field(name=
-        "---Morph's Auto Octo Finder---\n\nWords within {0} turns:\n".format(maxTurns), value="---------",inline=True)
+        "---Morph's Auto Octo Finder---\n\nWords within {0} turns:\nWarning, words starting with ! will not give the Wordsmith badge automatically. You will have to request for the word to be added in the #words channel\nResults are given with NewWord = NumberOfTurn NewWordScore\n".format(maxTurns), value="---------",inline=True)
         await channel.send(embed=header_embed)
         
         embed_results_string = str()
         turns = int()
         for pair in sortedResults:
-            turns = pair[1]
+            turns = pair[1][0]
             if turns <= maxTurns:
-                embed_results_string += str("{0} = {1}\n".format(pair[0],pair[1]))
-
+                if (client.wordsmith_words.find(i[0])!= -1):
+                    embed_results_string += str("{0} = {1} {2}\n".format(pair[0],pair[1][0],pair[1][1]))
```

Comment:
Can use "turns" instead of pair[1][0] - Helps with readability. I suggest we follow a similar pattern for the other indexed variables.